agenta-web/src/components/EvaluationTable/SimilarityMatchEvaluationTable.tsx

```diff
@@ -183,6 +184,7 @@ const SimilarityMatchEvaluationTable: React.FC<SimilarityMatchEvaluationTablePro
                 evaluation.evaluationType,
             )
                 .then((data) => {
+                    setRowValue(rowNumber, "similarity", similarity)
```

Comment:
`setRowValue` is called consecutively here, causing one of these 2 calls to be overwritten by another. Please see which of the `setRowValue` you want to keep here.

Reply:
Done. I took out the first.